ehitchhiking-ui/src/app/shared/components/rating/rating.component.html

```diff
@@ -0,0 +1,16 @@
+<fieldset class="rating">
+  <input type="radio" [name]="inputName" value="5" [checked]="rating===5" />
```

Comment:
Is it possible to use ngFor here ?

Reply:
stars counts decrease, no guesses how i should realize it

Reply:
It is possible, just extend used data model. *ngFor works with array of objects too.
For example, `const ratings = [ {title: 'Rocks!, value: 5}, ... ]`

Don't forget to create class or interface for entity. You should not use any everywhere